test-complete/nodejs-dmsdk-QueryToRdAll.js

```diff
@@ -114,40 +117,43 @@ describe('ReadAll with Snapshot and Update doc during read', function() {
         jsonDocreadable.push(null);
         dbWriter.documents.writeAll(jsonDocreadable, {
             onCompletion: ((summary) => {
-                setTimeout(()=>{var i = 0; i++;}, 1000);
+                setTimeout(() => { var i = 0; i++; }, 1000);
                 summary.docsWrittenSuccessfully.should.be.greaterThanOrEqual(100);
             })
```

Comment:
Since we changed line 105 from 100 to 500(uriCount), we can change line 121 as well.